modules/gdscript/gdscript_editor.cpp

```diff
@@ -2749,10 +3030,23 @@ static void _find_enumeration_candidates(GDScriptParser::CompletionContext &p_co
 
 		List<StringName> enum_constants;
 		ClassDB::get_enum_constants(class_name, enum_name, &enum_constants);
+
+		// Build hashmap of enum constants and whether or not they're deprecated
+		ERR_FAIL_COND_MSG(!class_list.has(class_name), vformat("%s class couldn't be found in doc data by the language server", class_name));
+		HashMap<String, bool> const_is_deprecated;
+		for (const DocData::ConstantDoc &constant : class_list.get(class_name).constants) {
+			const_is_deprecated.insert(constant.name, constant.is_deprecated);
+		}
 		for (const StringName &E : enum_constants) {
 			String candidate = class_name + "." + E;
 			int location = _get_enum_constant_location(class_name, E);
+			// TODO: Determine what case this covers.
```

Comment:
Native enum values for a context which has a native enum type.

Reply:
Is this a case where it would cannot currently be triggered, but would be if/when this method is called on Script types as well as native methods? And/or do you have an example of GDScript code that would trigger this completion branch?

Reply:
This can be triggered given the following script:
```gdscript
extends Control

func _init():
    set_anchors_and_offsets_preset(...)
```

Reply:
Thanks! Example has been added 😄